Ical.Net/DataTypes/IDateTime.cs

```diff
@@ -115,9 +115,27 @@ public interface IDateTime : IEncodableDataType, IComparable<IDateTime>, IFormat
     /// The returned <see cref="Value"/> is unchanged and the <see paramref="otherTzId"/> is set as <see cref="TzId"/>.
     /// </summary>
     IDateTime ToTimeZone(string otherTzId);
-    IDateTime Add(TimeSpan ts);
-    IDateTime Subtract(TimeSpan ts);
-    TimeSpan Subtract(IDateTime dt);
+
+    /// <summary>
+    /// Add the specified <see cref="Duration"/> to this instance/>.
+    /// </summary>
+    /// <remarks>
+    /// In correspondence to RFC5545, the weeks and day fields of a duration are considered nominal durations while the time fields are considered exact values.
+    /// </remarks>
+    IDateTime Add(Duration d);
+
+    /// <summary>Returns a new <see cref="TimeSpan" /> from subtracting the specified <see cref="IDateTime"/> from to the value of this instance.</summary>
```

Comment:
A xmldoc hint for SubtractExact vs Subtract similar like in static Duration FromTimeSpan(TimeSpan t) might be useful.